lib/color_spinor_pack.cu

```diff
@@ -377,13 +377,18 @@ namespace quda {
                        QUDA_FLOAT4_FIELD_ORDER>(ghost, a, parity, nFace, dagger, destination);
 
     } else if (a.FieldOrder() == QUDA_SPACE_SPIN_COLOR_FIELD_ORDER) {
+#if 0 // This doesn't work
       if (typeid(Float) != typeid(typename non_native_precision_mapper<Float>::type))
         errorQuda("Precision %d not supported for field type %d", a.Precision(), a.FieldOrder());
       if (typeid(ghostFloat) != typeid(typename non_native_precision_mapper<ghostFloat>::type))
         errorQuda("Ghost precision %d not supported for field type %d", a.GhostPrecision(), a.FieldOrder());
       genericPackGhost<typename non_native_precision_mapper<Float>::type,
                        typename non_native_precision_mapper<ghostFloat>::type,
                        QUDA_SPACE_SPIN_COLOR_FIELD_ORDER>(ghost, a, parity, nFace, dagger, destination);
+#else // The following works
+      genericPackGhost<Float, ghostFloat, QUDA_SPACE_SPIN_COLOR_FIELD_ORDER>(ghost, a, parity, nFace, dagger,
+                                                                             destination);
+#endif
```

Comment:
I am not familiar with the `non_native_precision_mapper` types here and I am not sure if the changes here affect other things. We should probably let @maddyscientist comment before removing it.